### PR TITLE
LPS-146233 Have remote apps upgrade tests run on other databases

### PR DESCRIPTION
**Pull request by susanchen3**
**Background**
Add ci properties to the RemoteAppsUpgrade.testcase file to make sure they are included in those test runs to better catch database issues with remote apps earlier. ie: [LPS-146200](https://issues.liferay.com/browse/LPS-146200)

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-146233

### CODE REVIEW

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
Just started reviewing :)

File: test.properties [hunk 1]
```diff
@@ -3763,6 +3772,33 @@
...
+        (testray.main.component.name == "Upgrades Remote Apps")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-sqlserver2017-jdk8][frontend]=\
+        (database.types ~ sqlserver) AND \
+        (testray.main.component.name == "Upgrades Remote Apps")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-sybase160-jdk8][frontend]=\
+        (database.types ~ sybase) AND \
+        (testray.main.component.name == "Upgrades Remote Apps")
+
```
Comment (john-co) [inline test.properties:3801]: I was wondering if our PQL is smart enough and this section could be simplified to something like the following and do the same thing:
```
test.batch.run.property.query[functional-upgrade-*][frontend]=\
        (testray.main.component.name == "Upgrades Remote Apps")
```

**Review by john-co — APPROVED** [review]
Looks good to me! :sunglasses:

**Comment by john-co** [comment]
ci:forward

File: test.properties (hunk 1 above)
Comment (susanchen3) [reply test.properties:3801]: ohhh, I can test it out

**Pull request closed without merging**